src/tags/kick.js

```diff
@@ -42,21 +42,9 @@ module.exports =
                 true,
                 noPerms
             );
-            console.debug(state);
-            switch (state) {
-                case 0: //Successful
-                    return 'Success';
-                case 1: //Bot doesnt have perms
-                    return error(`I don't have permission to kick users!`);
-                case 2: //Bot cannot kick target
-                    return error(`I don't have permission to kick ${user.username}!`);
-                case 3: //User doesnt have perms
-                    return error(`You don't have permission to kick users!`);
-                case 4: //User cannot kick target
-                    return error(`You don't have permission to kick ${user.username}!`);
-                default: //Error occurred
-                    throw state;
-            }
+            if (typeof response[1] == 'string' && response[1].startsWith('`'))
+            return Builder.util.error(subtag, context, response[1]);
```

Comment:
This line needs to be indented once again so that the return is part of the if scope and that we can access the line 47 in case we have no error.

Reply:
Indents dont matter for js, but yes this should be indented for style reasons